src/views/waypoint/home/components/get-started/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface GetStartedProps {
+  iconSvg: string
+  heading: string
+  text: string
+  link: {
+    url: string
+    text: string
+  }
+}
```

Comment:
Do you prefer to put component props interfaces in separate files than the component?

Reply:
No strong preference, have seen a separate `types.ts` file a few places. Happy to move this inline in the file if that feels more conventional 👍

Reply:
Feel free to leave it for now, I'll add it as an agenda item for our code style jam next week.